examples/workdir/inventory/hosts.yml

```diff
@@ -18,3 +18,23 @@ osh-deployer:
 ses_nodes:
   vars:
     ansible_user: root
+
+# added for airship
+airship-openstack-compute-workers:
+
+airship-openstack-control-workers:
+
+airship-ucp-workers:
```

Comment:
Why not reusing caasp workers?

Reply:
This can be a subset of the total available caasp workers.

Reply:
LGTM then